opengrok-indexer/src/main/java/org/opengrok/indexer/Metrics.java

```diff
@@ -45,8 +131,23 @@
     private Metrics() {
     }
 
-    public static PrometheusMeterRegistry getRegistry() {
-        return registry;
+    public static PrometheusMeterRegistry getPrometheusRegistry() {
+        return prometheusRegistry;
+    }
+
+    private static StatsdMeterRegistry getStatsdRegistry() {
+        return statsdRegistry;
     }
 
+    /**
+     * Get registry based on running context.
+     * @return MeterRegistry instance
+     */
+    public static MeterRegistry getRegistry() {
+        if (RuntimeEnvironment.getInstance().isIndexer()) {
+            return Metrics.getStatsdRegistry();
+        } else {
+            return Metrics.getPrometheusRegistry();
```

Comment:
Why use `Metrics.` prefix if the methods are in the same class?

Reply:
fixed